app/models/user.rb

```diff
@@ -19,6 +19,7 @@ class User < ApplicationRecord
   STREAMING_PLATFORMS = %w[twitch].freeze
   THEMES = %w[default night_theme pink_theme minimal_light_theme ten_x_hacker_theme].freeze
   TWITCH_URL_REGEXP = /\A(http(s)?:\/\/)?(www.twitch.tv|twitch.tv)\/.*\z/.freeze
+  USERNAME_MAX_LENGTH = 30
```

Comment:
Do any users in production have a username length greater than this that we need to fix to ensure they are not rendered invalid?

Reply:
Confirmed with @rhymes no users in production have a username longer than 30 characters